build.fsx

```diff
@@ -94,14 +94,20 @@ let setParams defaults = {
             ]
     }
 
+let Exec command args =
+    let result = Shell.Exec(command, args)
+    if result <> 0 then failwithf "%s exited with error %d" command result
+
 Target "BuildApp" (fun _ ->
     build setParams "./Octokit.sln"
         |> DoNothing
 )
 
-Target "BuildXSApp" (fun _ ->
-    build setParams "./Octokit-XamarinStudio.sln"
-        |> DoNothing
+Target "BuildMono" (fun _ ->
+    // xbuild does not support msbuild  tools version 14.0 and that is the reason
+    // for using the xbuild command directly instead of using msbuild
+    Exec "xbuild" "./Octokit-Mono.sln /t:Build /tv:12.0 /v:m  /p:RestorePackages='False' /p:Configuration='Release' /logger:Fake.MsBuildLogger+ErrorLogger,'../octokit.net/tools/FAKE.Core/tools/FakeLib.dll'"
```

Comment:
I tried it and it doesn't work.

>Building project: ./Octokit-Mono.sln
  xbuild  ./Octokit-Mono.sln /t:Build /tv:14.0 /v:m  /p:RestorePackages="False" >/p:Configuration="Release" >/logger:Fake.MsBuildLogger+ErrorLogger,"/home/travis/build/naveensrinivasan/octokit.net/tools/FAKE.Core/tools/FakeLib.dll"
>xbuild  ./Octokit-Mono.sln /t:Build /tv:14.0 /v:m  /p:RestorePackages="False" >/p:Configuration="Release" >/logger:Fake.MsBuildLogger+ErrorLogger,"/home/travis/build/naveensrinivasan/octokit.net/tools/F>>AKE.Core/tools/FakeLib.dll"
>XBuild Engine Version 12.0
Mono, Version 4.2.1.0
>Copyright (C) 2005-2013 Various Mono authors
>MSBUILD: error MSBUILD0000: Unknown tools version: '14.0' . Known versions: '2.0' '3.0' '3.5' '4.0' '12.0'
>Running build failed.
>Error:
>Building ./Octokit-Mono.sln failed with exitcode 1.

https://travis-ci.org/naveensrinivasan/octokit.net/jobs/96853565